plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -239,6 +239,16 @@ class ServicesTable extends React.Component {
     const scaleTextID = isGroup
       ? ServiceActionLabels.scale_by
       : ServiceActionLabels[SCALE];
+    const restartObj = {
+      className: classNames({
+        hidden: isPod || isGroup || instancesCount === 0
+      }),
+      id: RESTART,
+      html: this.props.intl.formatMessage({
+        id: ServiceActionLabels[RESTART]
+      })
+    };
+    const isSDK = isSDKService(service) ? [] : [restartObj];
```

Comment:
I think we should take this opportunity to clean up this method. We should remove all the `className` properties and follow the same pattern of appending actions to an array as we do in `ServiceDetail`.